talk/owt/BUILD.gn

```diff
@@ -297,25 +296,29 @@ static_library("owt_sdk_base") {
       "sdk/base/win/video_capture_mf.h",
       "sdk/base/win/videorendererd3d11.cc",
       "sdk/base/win/videorendererd3d11.h",
-      "sdk/base/win/msdkvideobase.cc",
-      "sdk/base/win/msdkvideobase.h",
     ]
+    if (owt_msdk_header_root != "") {
```

Comment:
Could you merge this to source set above?

Reply:
Moved above source set down here as their paths show they're for Windows only. But I haven't tested owt with MSDK yet.